e2e/tests/admin/user-admin-ui.spec.ts

```diff
@@ -0,0 +1,126 @@
+    
+    expect(columnNames.sort()).toEqual(columns.sort());
+
+    const usernameColIndex = await adminTable.getColumnIndex('User name');
+
+    //verify that the status col has status as active and user lockout displays ENABLE
+    const userNameValue = await userAdminPage.getUserNameText(1, usernameColIndex);
+
+    // validate that search box result 
+    expect(userNameValue).toEqual(userEmail);
+
+    // get the index of the Bypass column
+    const bypassColIndex = await adminTable.getColumnIndex('Bypass');
+
+    //click on the bypass link
+    const bypassPopup = await userAdminPage.clickBypassLink(1, bypassColIndex);
+
+    // Verify toggle option names match.
+    const toggleOptions = [
+      'Compliance Training',
+      'eRA Commons Linking',
+      'Two Factor Auth',
+      'Data Use Agreement',
+    ];
+    
+    const toggleTexts = await bypassPopup.getAllToggleTexts();
+    console.log(toggleTexts);
+
+    expect(toggleTexts.sort()).toEqual(toggleOptions.sort());
+
+    const trainingBypassToggle = bypassPopup.getTrainingBypassToggle();
+    expect(await trainingBypassToggle.isChecked()).toBe(true);
+
+    const eraCommBypassToggle = bypassPopup.getEraCommBypassToggle();
+    expect(await eraCommBypassToggle.isChecked()).toBe(true);
+
+    const twoFABypassToggle = bypassPopup.getTwoFABypassToggle();
+    expect(await twoFABypassToggle.isChecked()).toBe(true);
+
+    const dUCCBypassToggle = bypassPopup.getDUCCBypassToggle();
+    expect(await dUCCBypassToggle.isChecked()).toBe(true);
+    
+
+    //get the index of the user lockout column
+    const userLockoutColIndex = await adminTable.getColumnIndex('User Lockout');
+    await userAdminPage.getUserLockoutText(1, userLockoutColIndex);
+    const userLockout = await userAdminPage.getUserLockoutText(1, userLockoutColIndex);
+
+    const statusColIndex = await adminTable.getColumnIndex('Status');
+
+    //verify that the status col has status as active and user lockout displays ENABLE
+    const status = await userAdminPage.getStatusText(1, statusColIndex);
+
+    // switch statement to validate that the status reflects the user lockout 
+    switch(userLockout) {
+      case 'DISABLE':  
+        expect(status).toEqual('Active');
+        break;
+      case 'ENABLE':  
+      expect(status).toEqual('Disabled');
+        break;
+    }
+
+    //the name column index found separately since it is a frozen column
+    const nameColIndex = await adminTable.getNameColindex(); 
+
+    //click on the name link to navigate to the user profile page
+    const userProfileInfo = await userAdminPage.clickNameLink(1, nameColIndex);
+    await userProfileInfo.waitForLoad();
+    await page.bringToFront();
+
+    // click the audit link of the user
+    const auditColIndex = await adminTable.getColumnIndex('Audit');
+    console.log(`auditColIndex: ${auditColIndex}`);
+
+    const userAuditPage = await userAdminPage.clickAuditLink(1, auditColIndex);
+     await userAuditPage.waitForLoad();
```

Comment:
This time the test passed and the correct titles were read for both the new tabs. (userProfile-page & userAudit-page).